packages/core/src/components/toggletip/toggletip.tsx

```diff
@@ -0,0 +1,83 @@
+import { h, Component, Fragment, Element } from "@stencil/core";
+
+@Component({
+  tag: "dso-toggletip",
+  styleUrl: "toggletip.scss",
+  shadow: true,
+})
+export class Toggletip {
+  @Element()
+  host!: HTMLElement;
+
+  button!: HTMLButtonElement;
+
+  tooltip!: HTMLDsoTooltipElement;
+
+  componentDidRender() {
+    const button = this.host.shadowRoot?.querySelector("button");
+    if (!button) {
+      throw Error("button not found");
+    }
+
+    this.button = button;
+
+    const tooltip = this.host.shadowRoot?.querySelector("dso-tooltip");
+    if (!tooltip) {
+      throw Error("tooltip not found");
+    }
+
+    this.tooltip = tooltip;
+  }
+
+  click = () => {
+    this.tooltip.active ? this.close() : this.open();
+  };
+
+  open = () => {
+    this.tooltip.active = true;
```

Comment:
Kan je dit ook niet in de JSX verwerken?

Reply:
Huh?

Reply:
ja, ook aangepast